testing/smoketest/main.tf

```diff
@@ -44,7 +44,7 @@ resource "aws_iam_role" "lambda" {
 
 data "archive_file" "lambda" {
   type        = "zip"
-  source_file = "../testdata/function/index.js"
+  source_file = "./function/index.js"
```

Comment:
Can we reuse https://github.com/elastic/apm-aws-lambda/tree/main/testing/tf-modules/lambda_deployment instead of having a separate custom module for smoketest?